packages/core/src/components/banner/banner.tsx

```diff
@@ -11,11 +11,30 @@ export class Banner {
    * The status of the banner.
    */
   @Prop({ reflect: true })
-  status!: "warning" | "danger" | "error";
+  status!: "danger" | "error" | "info" | "warning";
+
+  /**
+   * compact mode.
+   */
+  @Prop()
+  compact = false;
+
+  /**
+   * *Only available when `compact` is set to `true`.
+   *
+   * Option to show banner icon.
+   */
+  @Prop()
+  noIcon = true;
```

Comment:
Wat zou je ervan vinden als we hier `icon = true` van maken? Ik begrijp dat het niet mogelijk is om defaults in de template models te definieren, maar ik vind dat we de API nu wel heel onlogisch maken 😕 

Als je het 'n goed idee vind mag je wat mij betreft de Angular templates ontmantelen, dan hoef je het daar niet bij te werken.

Reply:
Ik had het zo gedaan om het niet breaking te maken (html/css). In het webcomponent kunnen we er natuurlijk wel `icon = true` van maken zonder gevolgen. maar hoe willen we dat in html/css verwerken?

Reply:
Ik begrijp het. Ik denk dat we het zo laten, je hebt het mooi gemaakt.